Walls/VoidstoneWall.cs

```diff
@@ -12,15 +12,31 @@ namespace CalamityMod.Walls
     public class VoidstoneWall : ModWall
     {
         internal static Texture2D GlowTexture;
+        internal static bool[,] GlowTexture_NonEmptyFrame = new bool[13, 5];
+
 
         public override void SetStaticDefaults()
         {
             Main.wallHouse[Type] = true;
             GlowTexture = ModContent.Request<Texture2D>("CalamityMod/Walls/VoidstoneWall_Glowmask", AssetRequestMode.ImmediateLoad).Value;
 
+            // Hardcoded way to filter the Glowing frames
+            // Potentially be replaced to loop which look for actual pixel in certain frame
+            Array.Clear(GlowTexture_NonEmptyFrame);
+            GlowTexture_NonEmptyFrame[7, 2] = true;
+            GlowTexture_NonEmptyFrame[10, 0] = true;
+            GlowTexture_NonEmptyFrame[11, 1] = true;
```

Comment:
What loop would accomplish this? Also, is it necessary to use a cache array at all? Can't you just check these three specific values independently?